### PR TITLE
[Docs] small fixes

### PR DESCRIPTION
**Pull request by jtenner**
I noticed a few small problems with the docs. @willemneal can you approve this quickly?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -586,7 +586,7 @@ Regardless of the installed version, all the compiler flags will be passed to
 the `asc` command line tool.
 
 ```ts
-import asc from "assemblyscript/cli/asc";
+const compiler = require(require.resolve(path.join(options.compiler, "cli/asc"));
```
Comment (willemneal) [inline README.md]: Isn't it `path.resolve`?

**Comment by jtenner** [comment]
Right. Actually it's path.join(cwd(), compiler)

File: README.md (hunk 1 above)
Comment (jtenner) [reply README.md]: Yeah it's path.join(cwd(), options.compiler)

Comment (jtenner) [reply README.md]: This should be all set.

**Pull request merged**